dashi/bcl2fastq/index_summary.py

```diff
@@ -1,19 +1,21 @@
-import gsiqcetl.bcl2fastq.parse
+import gsiqcetl.bcl2fastq.cache
 import gsiqcetl.bcl2fastq.utility
 import dash_core_components as dcc
 import dash_html_components as html
 import dash.dependencies as dep
-import pandas
 
-index = gsiqcetl.bcl2fastq.parse.load_cache(
-    gsiqcetl.bcl2fastq.parse.CACHENAME.SAMPLES, "./data/bcl2fastq_cache.hd5"
+
+index = gsiqcetl.bcl2fastq.cache.load_known_samples_cache(
+    "v1", "./data/bcl2fastq_known_samples_cache.hd5"
```

Comment:
I see that some of the caches are "v1" and others are "v2". It's not clear from the gsi-qc-etl repository which modules have multiple versions, and what the differences between versions are. Could that be added somewhere, either in the README or linked prominently from the README?

Reply:
I have replaced the string by the gsi-qc-etl provided constants. Documentation of each schema is currently in the `columns` modules of gsi-qc-etl.